talent/application-process.md

```diff
@@ -4,27 +4,38 @@ This document describes the source{d} hiring process from the point of view of t
 
 ### Visiting our careers page
 
-- All the open roles can be found at https://sourced.tech/company/#careers where the developer is able to apply to any desired role.
+- All the open roles can be found at https://sourced.tech/company/#careers where the candidate is able to apply to any desired role.
 
-- If there isn't a role that fits but the developer is interested in working with us, they can send an email to talent@sourced.tech with their GitHub account and a small intro about themselves.
+- If there isn't a role that fits but the candidate is interested in working with us, they can send an email to talent@sourced.tech with their CV ( or GitHub account for developers) and a small intro of themselves.
 
 ### Screening
 
-- Less than a week after the submission of the application the developer receives an email with feedback; however there can be delays. In any case, the developer is free to reach out to us at any moment at talent@sourced.tech.
+- Less than a week after the submission of the application the candidate receives an email with an update; however there can be delays. In any case, the candiate is free to reach out to us at any moment at talent@sourced.tech.
 
-- In the case of positive feedback in the application, the developer is asked to move to a quick Skype/Zoom/Phone call with the lead of the potential hiring team.
+- In the case of positive feedback in the application, the candidate is invited to a quick video call with the Talent Acquisition Manager.
 
-### Skype/Zoom/Phone call
+### Initial interview: 1st stage
 
-- The developer has a quick call with the lead engineer of the team they applied to. This should be a general discussion that can last between 15-30 minutes. Its purpose is to get to know the background and experience of the developer, as well as allowing the developer to get to know source{d} better. It's expected that the developer has already done some basic research about source{d} and has specific questions about the role and/or the company to be expressed during the call. 
+- The candidate has an introductory video call with the Talent Acquisition Manager. This is a general discussion that can last between 15-30 minutes. This is to allow us to get to know the background and experience of the candidate, as well as allowing them to get to know source{d} better. It's expected that the candidate will have already done some basic research about source{d} and has some specific questions about the role and/or the company to be expressed during the call. 
```

Comment:
```suggestion
- The candidate has an introductory video call with the Talent Acquisition Manager. This is a general discussion that can last between 15-30 minutes. This is to allow us to get to know the background and experience of the candidate, as well as allowing them to get to know source{d} better. It's expected that the candidate has already done some basic research about source{d} and has some specific questions about the role and/or the company to be expressed during the call. 
```

Either: "has done … and has some specific questions …" or "will have done … and have specific questions"